ChangeLog.d/getrandom.txt

```diff
@@ -0,0 +1,2 @@
+Changes
```

Comment:
Commit message: please explain the reason of the changes done in the commit. I've read #3432 and the comments in it and and it is still not clear to me if this commit fixes a bug or/and improves the strength of the random generator.